Dockerfile

```diff
@@ -0,0 +1,169 @@
+FROM debian:buster
+
+RUN sed -i 's/main.*/main contrib non-free/' /etc/apt/sources.list
+
+RUN apt-get update && apt-get install -y \
```

Comment:
I think this is fine for now, but there are a couple paths we could ultimately go with:

- Use a common base Docker image for HT dependencies across applications
- Use Carton to manage perl dependencies

The former is probably easier given how we've managed these in the past, but longer-term we might want to think about managing dependencies more carefully with Carton.

Reply:
Yes, I'd like to use something like `hathitrust/feed_base:buster` from Docker Hub. IIRC I looked at trying to replace some of the CPAN recipes with Debian packages in the CRMS image, to speed up Travis builds. I came across an article referencing Carton and it seemed spiffy. If that's our future then I will pay more attention to it.